src/main/java/io/leonis/zosma/ipc/peripheral/component/xbox/RightClusterSupplier.java

```diff
@@ -0,0 +1,15 @@
+package io.leonis.zosma.ipc.peripheral.component.xbox;
+
+/**
+ * @author Jeroen de Jong
+ */
+public interface RightClusterSupplier {
```

Comment:
if this can be `RightClusterSupplier`, then why can't we have `LeftBumperSupplier`?

Reply:
You prefer 3 interfaces `LeftBumperSupplier`, `RightBumperSupplier` and `BumperSupplier` ?

Reply:
i think that would be a bit more consistent with how the rest is setup

Reply:
We could put it in a package :P ?

Reply:
that's what i meant by extracting the interfaces 😛